packages/core/src/utils/load-plugins.ts

```diff
@@ -14,22 +14,22 @@ export default function loadPlugin(
   [pluginPath, options]: [string, any],
   logger: ILogger
 ): IPlugin | undefined {
-  let plugin = tryRequire(pluginPath) as (
+  // tslint:disable-next-line:no-unnecessary-initializer
+  let plugin:
     | IPluginConstructor
-    | { default: IPluginConstructor });
+    | { default: IPluginConstructor }
+    | undefined = undefined;
 
-  // Try importing plugin as a path in CWD
-  if (!plugin) {
-    plugin = tryRequire(
-      path.join(process.cwd(), pluginPath)
-    ) as IPluginConstructor;
+  // Try requiring a path
+  if (pluginPath.startsWith('.') || pluginPath.startsWith('/')) {
+    plugin = tryRequire(pluginPath);
   }
 
-  // Try importing official plugin
-  if (!plugin) {
-    plugin = tryRequire(
-      path.join('@auto-it', pluginPath)
-    ) as IPluginConstructor;
+  // Try requiring a path from cwd
+  if (!plugin && (pluginPath.startsWith('.') || pluginPath.startsWith('/'))) {
+    plugin = tryRequire(path.join(process.cwd(), pluginPath));
+    logger.log.warn(`Could not find plugin from path: ${pluginPath}`);
+    return;
   }
 
   // For pkg bundle
```

Comment:
I'm generally _very_ uncertain about this. It's used for the binary I assume? It's _really_ hard to tell what the path this goes to matches with. We don't have tests around it either so it seems like it could fail silently in the future. 

Is there another way of approaching it?

Reply:
Yeah, this is awkward. 
I _think_ the intention was to be able to load one of the included plugins without having to hard-code a list of all of them somewhere, but we may have to go down that route (or create that list dynamically at build time) to make this more reproducible.

Reply:
It'd just be nice to have some sort of test to verify the paths so that if they change without updating this the tests let us know.